play-testkit/src/main/java/akka/grpc/play/JavaAkkaGrpcClientHelpers.java

```diff
@@ -14,16 +14,23 @@
 
 import javax.net.ssl.SSLContext;
 
+/** Helpers to test Java Akka gRPC clients with Play. */
 public final class JavaAkkaGrpcClientHelpers {
   private static final GrpcClientSettings$ GrpcClientSettings = GrpcClientSettings$.MODULE$;
 
   private JavaAkkaGrpcClientHelpers() {}
 
+  /** Creates a GrpcClientSettings from the given NewTestServer. */
   public static GrpcClientSettings grpcClientSettings(final NewTestServer testServer) {
     final ServerEndpoint http2Endpoint = unsafeGetHttp2Endpoint(testServer.endpoints());
     return grpcClientSettings(http2Endpoint, testServer.testServer().application().actorSystem());
   }
 
+  /**
+   * Unsafely gets the HTTP/2 endpoint from the given ServerEndpoints.
+   *
+   * If no HTTP/2 endpoint exists this throws an IllegalArgumentException.
+   */
```

Comment:
Why is this called `unsafeGet...` instead of a simple `get...`? I think in Java it's common to just call stuff `get...` and have exceptions thrown in case of failure.

Reply:
To try and communicate the lack of safety in calling it.

Reply:
we can improves this name later.